src/notify.c

```diff
@@ -107,12 +107,24 @@ void notifyKeyspaceEvent(int type, char *event, robj *key, int dbid) {
     robj *chanobj, *eventobj;
     int len = -1;
     char buf[24];
-
+    client *c = server.executing_client;
+    debugServerAssert((type & (NOTIFY_GENERIC | NOTIFY_STRING | NOTIFY_LIST | NOTIFY_SET | NOTIFY_HASH | NOTIFY_ZSET | NOTIFY_STREAM)) == 0 ||
+                 c == NULL ||
+                 c->cmd == NULL ||
+                 (c->cmd->flags & CMD_WRITE) == 0 ||
+                 c->flag.buffered_reply == 0 ||
+                 c->flag.keyspace_notified == 1 ||
+                 c->id == UINT64_MAX || // AOF client
+                 getClientType(c) != CLIENT_TYPE_NORMAL);
```

Comment:
Clang-format complains about this.

```suggestion
    debugServerAssert((type & (NOTIFY_GENERIC | NOTIFY_STRING | NOTIFY_LIST | NOTIFY_SET | NOTIFY_HASH | NOTIFY_ZSET | NOTIFY_STREAM)) == 0 ||
                      c == NULL ||
                      c->cmd == NULL ||
                      (c->cmd->flags & CMD_WRITE) == 0 ||
                      c->flag.buffered_reply == 0 ||
                      c->flag.keyspace_notified == 1 ||
                      c->id == UINT64_MAX || // AOF client
                      getClientType(c) != CLIENT_TYPE_NORMAL);
```

Reply:
Done